servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -757,8 +798,45 @@ public final Completable beforeCancel(Runnable onCancel) {
      * </ul>
      * for Subscriptions/{@link Subscriber}s of the returned {@link Completable}. <strong>MUST NOT</strong> throw.
      * @return The new {@link Completable}.
+     *
+     * @see <a href="http://reactivex.io/documentation/operators/do.html">ReactiveX do operator.</a>
      */
     public final Completable beforeFinally(Runnable doFinally) {
+        return beforeFinally(TerminalSignalConsumer.from(doFinally));
```

Comment:
Do we need to keep `beforeFinally(Runnable)`? Should we remove it and users can use `beforeFinally(TerminalSignalConsumer.from(doFinally))`  themselves.

Reply:
`Runnable` flavors are useful for state cleanups where type of termination does not matter. So, I will suggest to keep these methods.